packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md

```diff
@@ -253,3 +253,23 @@ If the middle initial is not empty, include it as part of the full name, using j
 
 If the user is a contractor and is a member of the "West Coast Users" user group, output "West coast contractors", else output "Others".<br>
 `user.profile.isContractor && user.isMemberOf({'group.profile.name': 'West Coast Users'}) ? "West coast contractors" : "Others"`
+
+## Security Expressions
+
+In [Okta Sign On Policies](/docs/reference/api/policy/#okta-sign-on-policy) and [App Sign On Policies](/docs/reference/api/policy/#app-sign-on-policy), you can additionally specify certain [Rule Conditions](/docs/reference/api/policy/#conditions) using expressions based on the security context of the request.
+The security context can be used to match requests for a given risk level or for matching a given behavior.
+
+Any Behavior defined in your organization can be used in a Rule Condition expression using the syntax `security.behaviors.contains('behaviorName')`.
+
+Examples:
+
+- `security.behaviors.contains('New Location')`
+- `security.behaviors.contains('New IP') && security.behaviors.contains('New Device')`
+
+Similarly, you can specify a Rule Condition matching a certain risk level by using the `security.risk.level` property, with any of the risk level values `LOW`, `MEDIUM`, or `HIGH`.
+Examples:
+
+- `security.risk.level == 'HIGH'`
+- `security.risk.level == 'MEDIUM'`
+
+
```

Comment:
nit: extra line.

Also, should we add a statement saying that these can be used in arbitrary combination with the supported operators e.g. `&&` `||` `!`? There is  already a section for what is supported [here](https://github.com/okta/okta-developer-docs/blob/595f9e72e59938f368b1e58dc6739dfc54eb8204/packages/%40okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md#constants-and-operators), maybe it can be reused

Reply:
I just added a third example here using such a combination.  I think that should suffice.